test/test_polar.jl

```diff
@@ -0,0 +1,180 @@
+@testset "Polar Decomposition" begin
+
+Random.seed!(1103)
+
+for T in [Float64, Complex{Float64}]
+    @testset "Newton $T" begin
+        for n in [1,3,10]
+
```

Comment:
Too many newlines here

Reply:
Ok, while I'm at it I'll make the test script more like the others